package.json

```diff
@@ -71,5 +71,14 @@
     "Alejandro (https://github.com/Minjung)",
     "Eli Skeggs <skeggse@gmail.com> (https://github.com/skeggse)",
     "Bent Cardan <bent@nothingsatisfies.com> (https://github.com/reqshark)"
+  ],
+  "files": [
+    "HISTORY.md",
+    "LICENSE",
+    "README.md",
+    "index.js",
+    "package.js",
+    "lib/",
+    "build/Release/zmq.node"
```

Comment:
If we don't include the `windows` directory, we won't be able to rebuild for Electron.